src/components/ConnectionsListProfiles.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import UserAvatar from "./UserAvatar";
+import { Button } from "antd";
+import { Profile } from "../utilities/types";
+
+enum ListStatus {
+  CLOSED,
+  FOLLOWERS,
+  FOLLOWING,
+}
+
+interface ConnectionsListProfilesProps {
+  listStatus: ListStatus;
+  connectionsList: Profile[];
+  notFollowing: Profile[];
```

Comment:
maybe we can call it `notFollowingProfiles`

Reply:
went for notFollowingList